### DISCUSSION TITLE
Game over when all tables are destroyed?

### DISCUSSION THREAD

**Discussion by smeas** [smeas/Beer-and-Plunder#186] [Ideas]
Currently we only have game over when the tavern's health reaches zero. Should we also have a trigger a game over if all tables are destroyed?

**Comment by tzaarela** [comment]
Yes, i was thinking of this earlier aswell.. the biggest issue i can see is that if you've reach the moneygoal and you still lose(because of tables) it might feel like the game robs you of the win.

**Reply by tzaarela** [reply]
You still have to repair the tables in order to survive economicly.. so i still Think you should lose some kind of amount of money when you choose to repair tables. But loosing all tables should still be viable if you make your moneygoal.. you still have to burn cash next round to repair... but then.. we start each round with 0 Gold right? Nothing gets kept from last round? Then how do we earn the first cash to repair the first table ?

**Comment by DreamingJohnny** [comment]
I think the main argument for having it be a lose condition to have all the tables be destroyed is that "game over" is that if that happens game over is probably inevitable in the long run.
And if that is the case, I think we should have game over happen earlier, do you understand what I'm saying?

So if it is unlikely that the players will come back from having all of the tables destroyed, then I think we should just end it when/if that happens.

**Comment by DreamingJohnny** [comment]
We decided "yes" so I'm locking this discussion.
